src/StatusIndicator/StatusIndicator.tsx

```diff
@@ -3,38 +3,48 @@ import styled from "styled-components";
 import { space, typography, flexbox, SpaceProps, TypographyProps, FlexboxProps } from "styled-system";
 import { DefaultNDSThemeType } from "../theme.type";
 
+export enum StatusIndicatorType {
```

Comment:
If we have to use an enum here, I would rename it to remove the `Type` keyword, since it can't be used with standard Typescript [utility types](https://www.typescriptlang.org/docs/handbook/utility-types.html).

Reply:
How do you like `StatusIndicatorView` instead?

Reply:
BTW, got propositions to use `as const` instead of `enum`. 
https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums